defaultConfig.json

```diff
@@ -26,6 +26,9 @@
         "request-token": "/plugins/servlet/oauth/request-token",
         "access-token": "/plugins/servlet/oauth/access-token",
         "authorize": "/plugins/servlet/oauth/authorize"
+      },
+      "stories": {
```

Comment:
Remove if not required for this task.

Reply:
This is required (jiraConfig.stories.regex). Please review further changes in the _test.js file (line 426)